.github/workflows/linux.yml

```
@@ -22,3 +22,72 @@ jobs:
         with:
           name: ooniprobe-linux-amd64
           path: ./CLI/linux/amd64/ooniprobe
+
+  armhf_build:
+    runs-on: "ubuntu-latest"
+    name: Build on armhf Buster
+    steps:
+      - uses: actions/checkout@v2.1.0
+      - uses: uraimo/run-on-arch-action@v2.0.5
+        name: Build ooniprobe
+        id: runcmd
+        with:
+          arch: armv7
+          distro: buster
+          # distro: ubuntu20.04 # no linux-headers-arm64
+
+          # Not required, but speeds up builds by storing container images in
+          # a GitHub package registry.
+          githubToken: ${{ github.token }}
+
+          run: |
+            uname -a
+            apt-get -qq update
+            apt-get install -qq -y --no-install-recommends gcc git linux-headers-armmp wget musl-dev ca-certificates libc6-dev
+            wget --no-verbose https://golang.org/dl/go1.16rc1.linux-armv6l.tar.gz
+            tar xfz go1.16rc1.linux-armv6l.tar.gz
+            export PATH=$PATH:$(pwd)/go/bin
+            go version
+            go env
+            go build -tags netgo -ldflags='-s -w -extldflags -static' ./cmd/ooniprobe
+
+      - name: Locate executable
+        run: find . -name ooniprobe -type f -executable
+
+      - name: Upload executable
+        uses: actions/upload-artifact@v1
+        with:
+          name: ooniprobe-linux-armhf
+          path: ooniprobe
+
+      - uses: uraimo/run-on-arch-action@v2.0.5
+        name: Build Debian package
+        with:
+          arch: armv7
+          distro: buster
+          githubToken: ${{ github.token }}
+          run: |
+            apt-get -qq update
+            #apt-get build-dep -y --no-install-recommends .
+            apt-get install -q  -y --no-install-recommends dpkg-dev build-essential devscripts debhelper
+            VER=$(./ooniprobe version)
+            if [[ ! $GITHUB_REF =~ ^refs/tags/* ]]; then
+              VER="${VER}~${GITHUB_RUN_NUMBER}"
+              dch -v $VER "New test version"
+              BT_REPO="${BT_REPO}-test"
+            else
+              dch -v $VER "New release"
+            fi
+            dpkg-buildpackage -us -uc -b
+            find ../ -name "*.deb" -type f
+            DEB="../ooniprobe-cli_${VER}_armhf.deb"
```

```suggestion
            DEB="../ooniprobe-cli_${VER}_armhf.deb"
            echo no | sudo dpkg -i $DEB
```

If someone edits the templates file and makes a mistake with the stanzas, the debian package will build but it will fail to install. This is not optimal. We should instead be failing the build here. The best way to do so, in my view, is to install the package and make sure the installation works.